configs/components/fesom/fesom-2.5.yaml

```
@@ -554,3 +557,20 @@ grids:
 further_reading:
         - fesom/fesom.forcing.yaml
 
+tidy_recipe:
+- "tidy_coupler"
+- "tar_binary_restarts"
+- "assemble"
+- "check_for_unknown_files"
+- "log_used_files"
+- "throw_away_some_infiles"
+- "copy_stuff_back_from_work"
+- "copy_all_results_to_exp"
+#- "_update_chunk_date_file"
+- "clean_run_dir"
+#- "start_post_job"
+#- "signal_tidy_completion"
+#- "_increment_date_and_run_number"
+#- "_write_date_file"
+#- "maybe_resubmit"
```

```suggestion
- "clean_run_dir"
```
This are remains of an older version of ESM-Tools